app/models/pipeline_run.rb

```diff
@@ -71,54 +83,51 @@ def finalized?
     finalized == 1
   end
 
+  def results_finalized?
+    results_finalized == 1
+  end
+
   def failed?
     /FAILED/ =~ job_status
   end
 
+  def initialize_result_status
+    # We explicitly set a value for result_status so that we can unambiguously
+    # identify pipeline_runs that predate result_status by the fact
+    # that their result_status is nil.
+    h = {}
+    target_outputs.each do |o|
+      h[o] = STATUS_EMPTY
+    end
+    self.result_status = h.to_json
+  end
+
   def create_run_stages
     # Host Filtering
     run_stages = []
     unless sample.host_genome && sample.host_genome.name == HostGenome::NO_HOST_NAME
```

Comment:
NO_HOST_NAME is no longer an option

Reply:
Actually, I can still see it in idseq.net/host_genomes. However, it was renamed to "No host subtraction __test__" a while ago. Changing that in the code

Reply:
The pipeline no longer supports it.   We always do at least ERCC.

Reply:
Ok. Will add an issue to make this change in a separate diff